cmd/user/main.go

```diff
@@ -39,11 +39,10 @@ func main() {
 
 	logger := log.New(config.Env.App.Environment)
 	eventStore := eventstore.New()
-	oauth2FacebookConfig := oauth2.NewConfigFacebook()
-	oauth2GoogleConfig := oauth2.NewConfigGoogle()
+	gothic.Store = sessions.NewCookieStore([]byte(config.Env.App.Secret))
```

Comment:
what do we store in session ? it was intended to be stateless api before,
does it mean now that we need middleware to fetch user from db

Reply:
> // Session needs to be implemented as part of the provider package.
> // It will be marshaled and persisted between requests to "tie"
> // the start and the end of the authorization process with a
> // 3rd party provider.

[goth session](https://github.com/markbates/goth/blob/master/session.go)